arte_downloader.sh

```diff
@@ -31,9 +31,9 @@ read -p "$(tput setaf 4)Choose quality (1-$NUMBER_OF_QUALITIES) $(tput sgr0)" CH
 SELECTED_FILE=$(echo $FILES | jq '.['$CHOSEN_QUALITY-1']')
 SELECTED_URL=$(echo $SELECTED_FILE | jq -r '.url')
 SELECTED_FORMAT=$(echo $SELECTED_FILE | jq -r '.mediaType')
-CLEANED_NAME=$(echo $TITLE | sed -e 's/[^A-Za-z0-9._-]/_/g')
+CLEANED_NAME=$(echo $TITLE | sed -e 's/[:\/]/_/g')
```

Comment:
```suggestion
CLEANED_NAME=$(echo $TITLE | sed -e 's/[^A-Za-z0-9._- "]/_/g')
```

Sorry for the delay. I'd be much happier with a whitelist of allowed chars. I added Space and Quotes. Any char you're missing?

Reply:
`(`, `)` and `&` for example, but it would also be nice to preserve german umlauts ... ;-)

I've modified the script to append the version/description, so choosing the fourth quality of
```sh
arte_downloader.sh https://www.arte.tv/de/videos/074493-001-A/ein-wunder-1-8/
```
results in
```sh
Saving file as /$PWD/Ein Wunder (1-8) [Deutsche Version - UT Hörgeschädigte].mp4...
```